content/providers/quick-start/data.yaml

```diff
@@ -340,85 +340,112 @@
       circuit = QuantumVolume(5)
+      circuit.measure_all()
 
-      # Transpile circuit
-      from qiskit import transpile
-      transpiled_circuit = transpile(circuit, backend)
-      transpiled_circuit.draw()
-    name: Transpile
+      # Select an execution backend
+      provider = AQTProvider("ACCESS_TOKEN")
+      backend = provider.get_backend("offline_simulator_no_noise")
+
+      # Transpile the circuit to target the selected backend
+      transpiled_circuit = qiskit.transpile(circuit, backend, optimization_level=2)
+      print(transpiled_circuit)
+
+      # Execute the circuit on the selected backend
+      result = backend.run(transpiled_circuit, shots=50).result()
+      print(result.get_counts())
+    name: Transpile and execute
     runMethod: backend
   - fullCode: |-
+      from qiskit import QuantumCircuit
+
       from qiskit_aqt_provider import AQTProvider
-      from qiskit.primitives import BackendSampler
-      aqt = AQTProvider('MY_TOKEN')
-      backend = aqt.get_backend('aqt_innsbruck')
-      sampler = BackendSampler(backend)
+      from qiskit_aqt_provider.primitives import AQTSampler
 
-      # Build circuit
-      from qiskit import QuantumCircuit
-      circuit = QuantumCircuit(2, 2)
+      # Define a circuit
+      circuit = QuantumCircuit(2)
       circuit.h(0)
-      circuit.cx(0,1)
-      circuit.measure([0,1], [0,1])
+      circuit.cnot(0, 1)
+      circuit.measure_all()
 
-      # Run the circuit and get result distribution
-      job = sampler.run(circuit)
-      quasi_dist = job.result().quasi_dists[0]
+      # Select an execution backend
+      provider = AQTProvider("ACCESS_TOKEN")
+      backend = provider.get_backend("offline_simulator_no_noise")
+
+      # Instantiate a sampler on the execution backend
+      sampler = AQTSampler(backend)
+
+      # Sample the circuit on the execution backend
+      result = sampler.run(circuit).result()
+
+      quasi_dist = result.quasi_dists[0]
       print(quasi_dist)
     name: Sample a Bell State
     runMethod: sampler
   - fullCode: |-
-      from qiskit_aqt_provider import AQTProvider
-      from qiskit.primitives import BackendEstimator
-      aqt = AQTProvider('MY_TOKEN')
-      backend = aqt.get_backend('aqt_innsbruck')
-      estimator = BackendEstimator(backend)
-
-      # Specify problem hamiltonian
-      from qiskit.quantum_info import SparsePauliOp
-      
-      hamiltonian = SparsePauliOp.from_list([
-         ("II", -1.052373245772859),
-         ("IZ", 0.39793742484318045),
-         ("ZI", -0.39793742484318045),
-         ("ZZ", -0.01128010425623538),
-         ("XX", 0.18093119978423156)
-      ])
-      
-      # Define VQE ansatz, initial point and cost function
       from qiskit.circuit.library import TwoLocal
+      from qiskit.quantum_info import SparsePauliOp
+      from scipy.optimize import minimize
+
+      from qiskit_aqt_provider import AQTProvider
+      from qiskit_aqt_provider.primitives import AQTEstimator
```

Comment:
In the old code, the imports are not all on the top because these code snippets are generated by a few pieces of codes together to scale up the generation for multiple algorithms x multiple providers (the scripts can be founder [here](https://github.com/HuangJunye/qiskit-providers-quickstart)). If functionally is the same, I think we should keep the same format across different providers.

Reply:
Thanks for the link! Does this also include infrastructure to test the examples and flag them if they get out of date?

Reply:
Adding testing infrastructure was the plan but it has been implemented and I am not sure when or whether it will be implemented because my job role has changed and I am no longer working on this.